src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -981,8 +981,7 @@ module.exports = function<T, P, I, TI, C, CX, CI>(config : HostConfig<T, P, I, T
     }
   }
 
-  function scheduleRoot(root : FiberRoot) {
-    const priorityLevel = root.current.pendingWorkPriority;
+  function scheduleRoot(root : FiberRoot, priorityLevel : PriorityLevel) {
```

Comment:
This is unfortunate that this was passed before but not accepted. How can we have caught that?